pocs/core.py

```diff
@@ -41,6 +41,7 @@ class POCS(PanStateMachine, PanBase):
     Attributes:
         name (str): Name of PANOPTES unit
         next_state (str): The next state for the state machine
+            TODO(jamessynge): Why document this here? It is an attribute of the base class.
```

Comment:
At one point I was going to document all the attributes of PanBase and PanStateMachine here so this is the only place you would have to look. That never happened and isn't a good idea, so this is a remnant.

Reply:
Removed next_state from docstring.